lib/aasm/base.rb

```diff
@@ -40,6 +42,9 @@ def initialize(klass, name, state_machine, options={}, &block)
       # Set to true to namespace reader methods and constants
       configure :namespace, false
 
+      # Configure a logger, with default being a Logger to STDERR
+      configure :logger, Logger.new(STDERR)
```

Comment:
I love small pull requests. So yes, it would be too much for this one. But I'm happy to see another pull request for your `AASM::Configuration` approach (there is already a class like this, which might be enhanced for this purpose).